_shared_content/automate/playbooks-on-premises.md

```diff
@@ -6,6 +6,8 @@ Clients must undertake a short installation process to harness the full potentia
 
 Below, we provide detailed instructions on how to accomplish the installation process.
 
+!!!note The Playbook runner supports only action, not trigger, execution on-premises.
```

Comment:
The call-out is malformed. The content should be on the next line
```suggestion
!!! note
     The Playbook runner supports only action, not trigger, execution on-premises.
```

The callout qualifier `note` is a bit sad. You can apply another qualifier according to the need: https://squidfunk.github.io/mkdocs-material/reference/admonitions/#supported-types

Reply:
Updated